pyproject.toml

```diff
@@ -26,15 +26,21 @@ classifiers = [
 ]
 license = "Apache 2.0"
 
+packages = [
+    { include = "meltano" },
+]
+
 [tool.poetry.urls]
 "Issue Tracker" = "https://github.com/meltano/edk/issues"
 "Twitter" = "https://twitter.com/meltanodata/"
 "Changelog" = "https://github.com/meltano/edk/blob/main/CHANGELOG.md"
 "Slack" = "https://meltano.com/slack"
 "Youtube" = "https://www.youtube.com/meltano"
 
+
+
 [tool.poetry.dependencies]
-ppython = "<3.11,>=3.7.1"
+python = "<3.11,>=3.7.1"
```

Comment:
Why `3.7.1` instead of `3.7.0`?

Reply:
Good catch, honestly not sure where/how that snuck in.